commands/drain.go

```diff
@@ -0,0 +1,53 @@
+package commands
+
+import (
+	"context"
+	"time"
+
+	"github.com/filecoin-project/dealbot/controller/client"
+	"github.com/urfave/cli/v2"
+)
+
+var DrainCmd = &cli.Command{
+	Name:   "drain",
+	Usage:  "Stop a dealbot daemon",
```

Comment:
I think you could make the usage text a bit clearer about this command waiting for tasks to finish, otherwise I'd assume the command would exit ~immediately, unless I was already familiar with the concept of "draining" :)